fs/config/config.go

```diff
@@ -63,6 +63,7 @@ type Config struct {
 	MaxConcurrency                   int64  `toml:"max_concurrency"`
 	NoPrometheus                     bool   `toml:"no_prometheus"`
 	PrioritizedTaskSilencePeriodMSec int    `toml:"prioritized_task_silence_period_msec"`
+	MountTimeout                     int    `toml:"mount_timeout"`
```

Comment:
We're kind of inconsistent about this, but we sometimes include the unit as a suffix for durations.

e.g. CacheConfig uses `_sec`
https://github.com/awslabs/soci-snapshotter/blob/main/fs/config/config.go#L81

but FuseCondig doesn't
https://github.com/awslabs/soci-snapshotter/blob/main/fs/config/config.go#L97

I think it makes sense to use the same convention everywhere and my preference would be to use the suffixes. What do you think?

Reply:
``MountTimeoutSec``? Sure